apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStoreParallelLiveTest.java

```diff
@@ -121,7 +126,7 @@ public void uploadMultipartBlob() {
 
    @Test(dependsOnMethods = "uploadMultipartBlob", singleThreaded = true)
    public void downloadParallelBlob() throws IOException {
-      final File downloadedFile = new File(BIG_FILE.getName() + ".downloaded");
+      final File downloadedFile = new File(BIG_FILE + ".downloaded");
```

Comment:
Not the case because each test method gets its own `BIG_FILE`.  Further these tests run single-threaded due to the annotation and the dependency.